lib/net.js

```diff
@@ -23,7 +23,7 @@ function setDebug(bool) {
 let peers = [];
 
 // The list of hardcoded, developer-approved seednodes
-const seednodes = ["144.91.87.251:8000", "164.68.102.142:45001", "164.68.115.32:45001", "45.77.128.182:9002"];
+const seednodes = ["144.91.87.251:8000", "164.68.102.142:45001", "164.68.115.32:45001", "45.77.128.182:9002, 89.40.5.16:9002"];
```

Comment:
IP needs to be in a seperate array string, seperated by a comma per-string as per previous nodes.